bin/ledger2beancount

```diff
@@ -53,6 +53,7 @@ $config->set_default(ledger_indent => 4);
 $config->set_default(beancount_indent => 2);
 $config->set_default(automatic_declarations => 1);
 $config->set_default(decimal_comma => 0);
+$config->set_default(ignore_virtual => 1);
```

Comment:
This look good, aside from the option name. It is a negative name (avoid doing something), which implies a confusing double negation when set to false. Can we make this something like "convert_virtual" or "handle_virtual" or whatever positive and have it default to false?

Reply:
Yeah, you're right.  I wasn't sure about the name.  I like your suggestion.